src/services/Token/index.ts

```diff
@@ -57,7 +57,7 @@ const TokenService: TokenService = {
     // -- END OVERRIDE -- //
     // Update listeners
     R.map(
-      (fn: Listener) => fn(value),
+      (fn: Listener) => setTimeout(R.partial(fn, [value]), 0),
```

Comment:
Silly person that wrote this before, making your work so much harder :P

Reply:
haha thanks for helping me with this bug